README.md

```diff
@@ -82,6 +82,30 @@ createTerminus(server, options);
 server.listen(PORT || 3000);
 ```
 
+Upon receiving the signal, Terminus will:
+
+1. Set the state to shutting down.
+   You can use it to start failing readiness probe.
+2. Execute and wait for `beforeShutdown`.
+   Here you can log the fact that shutdown has started.
+   You can also delay here shutting down HTTP server, so that the service is taken out from load balancer first.
```

Comment:
```suggestion
   If you perform asynchronous code, this will delay the shutting down of the HTTP server, but if you've configured your readiness probe correctly, your instance may be removed from your load balancer before this completes.
```